tests/e2e/test_rosacli_cluster.go

```diff
@@ -3067,3 +3063,105 @@ var _ = Describe("Sts cluster creation supplemental testing",
 				Expect(clusterID).ToNot(BeNil())
 			})
 	})
+
+var _ = Describe("User can set availability zones to create rosa multi-az STS cluster supplemental testing",
+	labels.Feature.Cluster,
+	func() {
+		defer GinkgoRecover()
+
+		var (
+			rosaClient     *rosacli.Client
+			clusterService rosacli.ClusterService
+
+			customProfile      *profilehandler.Profile
+			clusterID          string
+			ocmResourceService rosacli.OCMResourceService
+			testingClusterName string
+		)
+		BeforeEach(func() {
+
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+			clusterService = rosaClient.Cluster
+			ocmResourceService = rosaClient.OCMResource
+
+			By("Prepare custom profile")
+			customProfile = &profilehandler.Profile{
+				ClusterConfig: &profilehandler.ClusterConfig{
+					HCP:           false,
+					MultiAZ:       true,
+					STS:           true,
+					OIDCConfig:    "",
+					NetworkingSet: false,
+					BYOVPC:        false,
+					Zones:         "us-west-2a,us-west-2b,us-west-2c",
+				},
+				AccountRoleConfig: &profilehandler.AccountRoleConfig{
+					Path:               "/aa/bb/",
+					PermissionBoundary: "",
+				},
+				Version:      "latest",
+				ChannelGroup: "candidate",
+				Region:       "us-west-2",
+			}
+			customProfile.NamePrefix = constants.DefaultNamePrefix
+
+		})
+
+		AfterEach(func() {
+			By("Delete cluster")
+			rosaClient.Runner.UnsetArgs()
+			_, err := clusterService.DeleteCluster(clusterID, "-y")
+			Expect(err).To(BeNil())
+
+			rosaClient.Runner.UnsetArgs()
+			err = clusterService.WaitClusterDeleted(clusterID, 3, 30)
+			Expect(err).To(BeNil())
+
+			By("Delete operator-roles")
+			_, err = ocmResourceService.DeleteOperatorRoles(
+				"-c", clusterID,
+				"--mode", "auto",
+				"-y",
+			)
+			Expect(err).To(BeNil())
+
+			By("Clean resource")
+			errs := profilehandler.DestroyResourceByProfile(customProfile, rosaClient)
+			Expect(len(errs)).To(Equal(0))
+		})
+
+		It("User can set availability zones to create rosa multi-az STS cluster - [id:56224]",
+			labels.Critical, labels.Runtime.Day1Supplemental,
+			func() {
+				By("Create classic sts cluster in auto mode")
+				testingClusterName = "rosa56224"
+				flags, err := profilehandler.GenerateClusterCreateFlags(customProfile, rosaClient)
+				Expect(err).ToNot(HaveOccurred())
+
+				command := "rosa create cluster --cluster-name " + testingClusterName + " " + strings.Join(flags, " ")
```

Comment:
I don't understand the implementation here. Why not user rosaclient.ClusterService to create the cluster. This kind of the command cannot output the log. It will be very hard to debug in future with failure

Reply:
Updated using clusterService.Create() function to create cluster.